packages/core/src/common/extract/extractTaggedOutputs.ts

```diff
@@ -12,6 +12,7 @@ const BASE_AUTOPILOT_STEP = {
   screenDescription: { tag: "SCREENDESCRIPTION", isRequired: true },
   thoughts: { tag: "THOUGHTS", isRequired: true },
   action: { tag: "ACTION", isRequired: true },
+  summary: { tag: "SUMMARY", isRequired: true },
```

Comment:
this shouldn't be mandatory, the summary is for the last step only, and that's how we define it on the prompt